app/controllers/api/v1/scholarship_applications_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Api
+  module V1
+    class ScholarshipApplicationsController < ApplicationController
+      respond_to :json
+
+      def create
+        @scholarship_application = ScholarshipApplication.new(reason: params[:reason], terms_accpeted: params[:terms_accepted], scholarship_id: params[:scholarship_id], user_id: current_user.id)
```

Comment:
Since I don't send `:user_id` from the front, in other words since we don't receive it as a paramter, I went ahead and did this

```ruby
params.require(:scholarship_application).permit(:reason, :terms_accepted, :scholarship_id).merge(user_id: current_user.id)
```